timescaledb/getting-started/index.md

```diff
@@ -1,23 +1,50 @@
 # Getting started with TimescaleDB
 
-**Congratulations!** You're ready to jump in and start using TimescaleDB.
+**Congratulations!** You're likely here because you have successfully spun 
+up your first instance of TimescaleDB, can connect to it, and are ready to 
+explore some of the most popular TimescaleDB feautures. If you have not yet 
+created a TimescaleDB instance or learned how to connect to it, make sure to 
+first check out these two sections within the docs:
+ * [Install TimescaleDB][install]
+ * [Connecting to TimescaleDB][connecting]
+
 
 ## What is TimescaleDB?
-Developers often refer to TimescaleDB as PostgreSQL with time-series superpowers.
-Put another way, TimescaleDB is a relational database with native time-series
-optimizations to make ingesting, storing and analyzing time-series data at petabyte
-scale fast and easy. TimescaleDB is packaged as a PostgreSQL extension. It also
-supports all the standard PostgreSQL database objects, like tables, indexes,
-triggers, and more. Moreover, a TimescaleDB database is able to coexist with
-PostgreSQL databases (and other TimescaleDB databases) on a single server.
-
-TimescaleDB also supports the full SQL language and you can use TimescaleDB with
-all the tools and connectors in the PostgreSQL ecosystem. If it works with
-PostgreSQL, it works with Timescale!
-
-## Let's get up and running
+As you likely know already, TimescaleDB is an extension on top of the PostgreSQL. 
+When you use TimescaleDB, you get all the time-series optimizations and special 
+functionality that TimescaleDB provides, along with all the features available 
+in vanilla PostgreSQL. Developers often refer to TimescaleDB as PostgreSQL with 
```

Comment:
```suggestion
in vanilla PostgreSQL. Developers refer to TimescaleDB as PostgreSQL with 
```

Reply:
Edited Ryan's suggestion to say 'standard' instead of 'vanilla' (less metaphorical/not a cultural reference = more inclusive for readers of English as as second language)